agent/config/config_linux.go

```diff
@@ -13,24 +13,12 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package control
+package config
 
-import (
-	"github.com/aws/amazon-ecs-agent/agent/config"
+import "github.com/containerd/cgroups"
 
-	"github.com/cihub/seelog"
-	specs "github.com/opencontainers/runtime-spec/specs-go"
-)
-
-// Init is used to setup the cgroup root for ecs
-func (c *control) Init() error {
-	seelog.Infof("Creating root ecs cgroup: %s", config.DefaultTaskCgroupPrefix)
-
-	// Build cgroup spec
-	cgroupSpec := &Spec{
-		Root:  config.DefaultTaskCgroupPrefix,
-		Specs: &specs.LinuxResources{},
+func init() {
```

Comment:
Q. (I might miss it) where/when do we call this function to update `CgroupV2` flag? Thanks!

Reply:
in go the init() function is a special function that executes before any other functions in a package are executed, it's generally used for initializing things: https://golangdocs.com/init-function-in-golang

Reply:
Thanks!